Example/Messaging/Sample/Sample/ContentView.swift

```diff
@@ -123,15 +140,35 @@ struct DetailView: View {
 
   var body: some View {
     VStack {
-      Text("InstanceID: \(self.identity.instanceID)")
-      Text("Token: \(self.identity.token)")
+      Text("InstanceID: \(self.identity.instanceID ?? "None")")
+      Text("Token: \(self.identity.token ?? "None")")
     }
   }
 }
 
 struct ContentView_Previews: PreviewProvider {
+  // A fake filled identity for testing rendering of a filled cell.
+  static let filledIdentity: Identity = {
```

Comment:
oh this is really nice cuz I found rendering sometimes breaks for trying to get the real identity data

Reply:
Yeah it's really helpful being able to inject data like this :D